Session/Settings/HelpViewModel.swift

```diff
@@ -227,12 +227,23 @@ class HelpViewModel: SessionTableViewModel, NavigatableStateHolder, ObservableTa
                 let viewController: UIViewController = dependencies[singleton: .appContext].frontMostViewController
             else { return }
             
+            let filePath = URL(fileURLWithPath: latestLogFilePath)
+            
+            /// To not modify the existing files generated and modified via `Log.logFilePath`
+            /// only the file to be shared will be sanitized by removing whitespaces
+            let sanitizedFileURL = Log.prepareFileForSharing(originalURL: filePath)
+
             let showShareSheet: () -> () = {
                 let shareVC = UIActivityViewController(
-                    activityItems: [ URL(fileURLWithPath: latestLogFilePath) ],
+                    activityItems: [
+                        sanitizedFileURL
+                    ],
                     applicationActivities: nil
                 )
                 shareVC.completionWithItemsHandler = { _, success, _, _ in
+                    /// Deletes file copy of the log file
+                    Log.deleteItem(at: sanitizedFileURL)
```

Comment:
This can just be called on the `FileManagerType` singleton directly (elsewhere in the code I have a sanity check to ensure it's stored in the temporary directory before removing it just to make sure a future code change doesn't result in a bug where the original file gets removed)
```
/// Sanity check to make sure we don't unintentionally remove a proper attachment file
if path.hasPrefix(dependencies[singleton: .fileManager].temporaryDirectory) {
    try? dependencies[singleton: .fileManager].removeItem(atPath: path)
}
```